views/partials/_tracking-body.njk

```diff
@@ -1,4 +1,5 @@
+
 <!-- Google Tag Manager (noscript) -->
-<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-53XG2JT"
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{GTM_TAG}}"
```

Comment:
I don't think we have this written down anywhere, but I think we generally include a space on the inside of the double parens (`{{ GTM_TAG }}`).